src/Core/src/PropertyMapperExtensions.cs

```diff
@@ -145,5 +145,80 @@ void newMethod(IElementHandler handler, IElement view)
 				action?.Invoke(handler!, view);
 			});
 		}
+
+		/// <summary>
+		/// Used internally to provide a fluent API for easily modifying
+		/// mappers at the point of instantiation.
+		/// Specify a method to be run before an existing property mapping.
+		/// </summary>
+		/// <typeparam name="TVirtualView">The cross-platform type.</typeparam>
+		/// <typeparam name="TViewHandler">The handler type.</typeparam>
+		/// <param name="propertyMapper">The property mapper in which to change the mapping.</param>
+		/// <param name="key">The name of the property.</param>
+		/// <param name="method">The method to call before the existing mapping begins.</param>
+		/// <returns>Target Property Mapper</returns>
+		internal static IPropertyMapper<TVirtualView, TViewHandler> Modify<TVirtualView, TViewHandler>(this IPropertyMapper<TVirtualView, TViewHandler> propertyMapper,
+			string key, Action<TViewHandler, TVirtualView, Action<IElementHandler, IElement>?> method)
+			where TVirtualView : IElement where TViewHandler : IElementHandler
+		{
+			propertyMapper.ModifyMapping(key, method);
```

Comment:
The overloads are very easy to mistake, but this particular one, the one on `this IPropertyMapper<TVirtualView, TViewHandler> propertyMapper` (with the generic `TViewHandler`) does a straight replace:

```cs
public static void ModifyMapping<TVirtualView, TViewHandler>(this IPropertyMapper<TVirtualView, TViewHandler> propertyMapper,
	string key, Action<TViewHandler, TVirtualView, Action<IElementHandler, IElement>?> method)
	where TVirtualView : IElement where TViewHandler : IElementHandler
{
	var previousMethod = propertyMapper.GetProperty(key);

	void newMethod(TViewHandler handler, TVirtualView view)
	{
		method(handler, view, previousMethod);
	}

	propertyMapper.Add(key, newMethod);
}
```